### PR TITLE
Remove akka-contrib

### PR DESCRIPTION
**Pull request by raboof**
Refs #26183

This module has been deprecated since 2.5 and is removed in 2.6

### CODE REVIEW

**Review by patriknw — COMMENTED** [review]
Start a migration guide 2.5 -> 2.6 page also and mention the removals there. Here is not much migration guidance other than the source can be copied if someone is depending on it.

**Review by patriknw — COMMENTED** [review]
mention in migration guide, replacement would be to copy used sources from 2.5

**Review by chbatey — COMMENTED** [review]
LGTM

**Review by chbatey — APPROVED** [review]

File: akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Migration Guide 2.5.x to 2.6.x
 
+## Removed module: akka-contrib
+
+The akka-contrib module was deprecated in 2.5 and has been removed in 2.6.
```
Comment (patriknw) [inline akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md]: 2.5.0

**Review by patriknw — APPROVED** [review]
LGTM, but has conflict in migration guide now

**Comment by patriknw** [comment]
rebase and merge

**Pull request merged**